backend/api/parse_pdf.py

```
@@ -390,6 +746,15 @@ async def parse_pdf_fast_autofill(
         
     except Exception as e:
         debug_log(f"[FAST_AUTOFILL_ERROR] {str(e)}")
+        # If scenario was created, update status to indicate error
+        if scenario_id:
+            try:
+                scenario = db.query(Scenario).filter(Scenario.id == scenario_id).first()
+                if scenario:
+                    scenario.status = "draft"  # Set to draft so user can still access it
+                    db.commit()
+            except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)